ops/testing.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2021 Canonical Ltd.
```

Comment:
Year added = 2024?

Reply:
Yeah, it wasn't clear what to do with that. There is a file `ops/testing.py` that has existed since 2021. The content of the new file `ops/_private/harness.py` has existed since 2021. I wasn't sure what years to put in. If it's purely on the content then I suppose 2024 is more correct for `ops/testing.py` but it's also a bunch of imports, not really copyright-able anyway.

Happy to do this if you prefer:

```suggestion
# Copyright 2024 Canonical Ltd.
```

Reply:
Yep, I think the 2024 makes more sense as it's really a completely new file.